static_src/stores/service_instance_store.js

```diff
@@ -150,18 +161,27 @@ class ServiceInstanceStore extends BaseStore {
           action.spaceGuid,
           action.servicePlanGuid
         );
+        this._createLoading = true;
+        this.emitChange();
         break;
       }
 
       case serviceActionTypes.SERVICE_INSTANCE_CREATED: {
         cfApi.fetchServiceInstance(action.serviceInstance.guid);
-        this._createInstanceForm = null;
+        this._createLoading = false;
+        this._createdTempNotification = true;
         this.emitChange();
+        setTimeout(() => {
+          this._createInstanceForm = null;
+          this._createdTempNotification = false;
+          this.emitChange();
+        }, CREATED_NOTIFICATION_TIME_MS);
```

Comment:
I was thinking about this kind of time-based UI change when working on the quicklook loading and FWIW I think this should be done in the action. It would look something like this:
```javascript
instanceCreate(serviceData) {
  // dispatch SERVICE_INSTANCE_CREATE
  return cfApi
    .createServiceInstance(serviceData)
    .then((guid) => cfApi.fetchServiceInstance(guid).then(this.instanceCreated));
}

instanceCreated(instance) {
  // dispatch SERVICE_INSTANCE_CREATED
  setTimeout(/* dispatch SERVICE_INSTANCE_NOTIFICATION_CLEAR */);
}
```

What do you think?